docker-run.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+set -e
+
+IMAGE_NAME=test-redds
+
+while [[ $# -gt 0]]; do
+    case "$1" in 
+        --profile-cpu)
+            PROFILE_CPU=true
+            CPU_PROFILE_OUTPUT_PATH="$2"
+            shift 2;;
+        --drive-upload)
+            DRIVE_UPLOAD=true 
+
+            DRIVE_SERVICE_ACCOUNT_CREDENTIALS_URL=$2
+            MESSAGES_DRIVE_PATH=$3
+            INDIVIDUALS_DRIVE_PATH=$4
+            PRODUCTION_DRIVE_PATH=$5
+            shift 5;;
+        --)
+            shift
+            break;;
+        *)
+            break;;      
+        esac
+    done 
+
+#Check that the correct number of arguments were provided
+if [[ $# -ne 17 ]]; then 
+    echo "Usage: ./docker-run.sh 
+    [--profile-cpu <profile-output-path>] 
+    [--drive-upload <drive-auth-file> <messages-drive-path> <individuals-drive-path> <production-drive-path>]
+    <user> <phone-number-uuid-table-path> 
+    <internet_working_poll-input-path> <water_filter_poll-input-path> <waste_disposal-input-path> <demog-input-path> <prev-coded-dir>
```

Comment:
please be consistent with the '-'s vs '_'s here.

I believe hyphens are standard in unix tool argument names